es_ES/extension_settings.txt

```diff
@@ -40,24 +60,49 @@ extension_settings.followme = Sígueme
 extension_settings.fw_ringtime = Tiempo de Timbrado
 extension_settings.fw_ringtime.tooltip = Es el tiempo que el teléfono timbrará en la lista sígueme sin ser atendido, antes de continuar hacia un destino alternativo, tales como correo de voz.
 extension_settings.general = General
+extension_settings.greet = Mensaje de Saludo
+extension_settings.greet.invalid = El archivo proporcionado no parece ser un archivo de audio
+extension_settings.greet.tooltip = Mensaje de audio que se reproduce cuando el llamante alcanza el buzón de voz de esta extensión y no hay mensajes de No Disponible o Ocupado presentes
 extension_settings.no_extension = Usted no tiene una extensión para el usuario del portal.
+extension_settings.no_diversions = No se permiten desvíos
+extension_settings.no_file_selected = No se ha seleccionado ningún archivo de audio
+extension_settings.no_recording = Archivo de audio no encontrado
+extension_settings.notifications = Notificaciones por correo electrónico
+extension_settings.notifications.tooltip = Activa/desactiva notificaciones por correo electrónico del buzón de voz
 extension_settings.personal_assistant.info = El IVR del Asistente Personal no se activará hasta que un saludo sea grabado marcando %s.
 extension_settings.personal_assitant = Asistente Personal
 extension_settings.personal_assitant.tooltip = Desactivar no eliminará ninguna configuración existente de usuarios, sino que deshabilitará el acceso a la función.
 extension_settings.phone_features = Características del Teléfono
 extension_settings.press = Presione {0}
```

Comment:
I had reviewed the translation on the User Portal in Spanish but the translation doesn't apply, it still shows Press followed by the number instead of "Presione"